lms/templates/admin/index.html.jinja2

```diff
@@ -0,0 +1,16 @@
+{% extends "lms:templates/admin/base.html.jinja2" %}
+
+
+{% block content %}
```

Comment:
There's a loner input field here on the index page :shrug:

Reply:
I think maybe the `/admin/` page should just redirect to `/admin/installations/`, then it doesn't need a template. Even when we have more than one admin page in future, `/admin/` and still redirect to `/admin/installations/` and if people want a different page they can use the navbar at the top to navigate to where they want to go

Reply:
Yeah, I'll do that, current index.html it's pointless